### ISSUE TITLE
Reports ignoring the --ignore-missing-imports flag

### ISSUE THREAD

**Issue by fcanobrash** [python/mypy#8446]
Mypy report generation might be ignoring the flag `--ignore-missing-imports`.

Example:
Considering a file that imports a module without any typing information:
```python
from flask_httpauth import HTTPBasicAuth
...
```
If mypy is executed as follows:
`mypy --ignore-missing-imports --html-report reports myfile.py`
No error is raised in the command line because the missing import is correctly ignored, but in the generated report, mypy still shows an imprecision over that line.

Adding typing information for the module solves the red line in the report but shouldn't the report take the `--ignore-missing-imports` flag into account and not show a red line in this case?

**Comment by fcanobrash** [comment]
Sorry, this issue in not completely accurate. Closing, thanks.

**Comment by fcanobrash** [comment]
After some more testing, I think this issue is valid. Sorry for closing it too soon.

**Comment by JukkaL** [comment]
This is working as intended. `--ignore-missing-imports` causes `Any` types to be inferred, and the report will show them as such. If you want to get rid of them, you can try generating stubs using `stubgen`, for example. Another alternative is to define a fully annotated wrappers around libraries without stubs, so that the type checking imprecision is limited to the wrappers.

I might accept a PR that adds a `--html-report` option to ignore `Any` types from missing imports.
